spring-cloud-dataflow-platform-cloudfoundry/pom.xml

```diff
@@ -33,25 +33,13 @@
 		  <optional>true</optional>
 		</dependency>
 		<dependency>
-			<groupId>io.pivotal.spring.cloud</groupId>
-			<artifactId>spring-cloud-services-starter-config-client</artifactId>
-			<version>${spring-cloud-services.version}</version>
-			<scope>runtime</scope>
-			<exclusions>
-				<exclusion>
-					<groupId>org.springframework.cloud</groupId>
-					<artifactId>spring-cloud-cloudfoundry-connector</artifactId>
-				</exclusion>
-				<exclusion>
-					<groupId>org.springframework.cloud</groupId>
-					<artifactId>spring-cloud-spring-service-connector</artifactId>
-				</exclusion>
-			</exclusions>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-config</artifactId>
 		</dependency>
 		<dependency>
-			<groupId>io.pivotal.spring.cloud</groupId>
-			<artifactId>spring-cloud-sso-connector</artifactId>
-			<version>${spring-cloud-sso.version}</version>
+			<groupId>org.springframework.cfenv</groupId>
+			<artifactId>spring-cfenv-boot</artifactId>
```

Comment:
Think we need to rename module as we always get heat from boot guys if anyone else uses `boot` in naming.

Reply:
Maybe not necessarily right now but before we actually release something.